src/BotwinExtensions.cs

```diff
@@ -82,17 +83,20 @@ private static RequestDelegate CreateRouteHandler(string path, Type moduleType)
 
                 if (module.Before != null)
                 {
+                    logger?.LogTrace("Executing module before hook");
```

Comment:
Imho, if people want to be notified of their before/after handler being run, they should add logging calls inside of their modules themselves.

(same below with the after handler)